### PR TITLE
Update Distributed Tracing upstream jobs.

### PR DESCRIPTION
**Pull request by IshwarKanse**
* Update the jobs to sync with the latest upstream changes.

### CODE REVIEW

**Comment by IshwarKanse** [comment]
/pj-rehearse pull-ci-openshift-grafana-tempo-operator-main-tempo-e2e-tests pull-ci-openshift-open-telemetry-opentelemetry-operator-main-opentelemetry-e2e-tests

**Comment by IshwarKanse** [comment]
/pj-rehearse pull-ci-openshift-open-telemetry-opentelemetry-operator-main-opentelemetry-e2e-tests

**Comment by IshwarKanse** [comment]
/pj-rehearse pull-ci-openshift-grafana-tempo-operator-main-tempo-e2e-tests pull-ci-openshift-open-telemetry-opentelemetry-operator-main-opentelemetry-e2e-tests

**Comment by IshwarKanse** [comment]
/pj-rehearse periodic-ci-openshift-open-telemetry-opentelemetry-operator-main-ocp-4.13-amd64-upgrade-opentelemetry-operator-upgrade periodic-ci-openshift-grafana-tempo-operator-main-ocp4.13-amd64-upgrade-tempo-operator-upgrade

**Comment by IshwarKanse** [comment]
/pj-rehearse periodic-ci-openshift-open-telemetry-opentelemetry-operator-main-ocp-4.13-amd64-upgrade-opentelemetry-operator-upgrade periodic-ci-openshift-grafana-tempo-operator-main-ocp4.13-amd64-upgrade-tempo-operator-upgrade

**Comment by IshwarKanse** [comment]
/pj-rehearse periodic-ci-openshift-grafana-tempo-operator-main-ocp4.13-amd64-upgrade-tempo-operator-upgrade

**Comment by IshwarKanse** [comment]
/pj-rehearse periodic-ci-openshift-open-telemetry-opentelemetry-operator-main-ocp-4.13-amd64-upgrade-opentelemetry-operator-upgrade periodic-ci-openshift-grafana-tempo-operator-main-ocp4.13-amd64-upgrade-tempo-operator-upgrade

**Comment by IshwarKanse** [comment]
/pj-rehearse pull-ci-openshift-open-telemetry-opentelemetry-operator-main-opentelemetry-e2e-tests

**Comment by IshwarKanse** [comment]
/pj-rehearse pull-ci-openshift-open-telemetry-opentelemetry-operator-main-opentelemetry-e2e-tests periodic-ci-openshift-open-telemetry-opentelemetry-operator-main-ocp-4.13-amd64-upgrade-opentelemetry-operator-upgrade

**Comment by IshwarKanse** [comment]
/pj-rehearse periodic-ci-openshift-open-telemetry-opentelemetry-operator-main-ocp-4.13-amd64-upgrade-opentelemetry-operator-upgrade

**Comment by IshwarKanse** [comment]
/pj-rehearse periodic-ci-openshift-open-telemetry-opentelemetry-operator-main-ocp-4.13-amd64-upgrade-opentelemetry-operator-upgrade

**Comment by IshwarKanse** [comment]
/pj-rehearse periodic-ci-openshift-open-telemetry-opentelemetry-operator-main-ocp-4.13-amd64-upgrade-opentelemetry-operator-upgrade

File: ci-operator/config/openshift/open-telemetry-opentelemetry-operator/openshift-open-telemetry-opentelemetry-operator-main__ocp-4.14-amd64.yaml [hunk 1]
```diff
@@ -76,9 +78,9 @@ tests:
       BASE_DOMAIN: qe.devcluster.openshift.com
       EO_SUB_CHANNEL: stable-5.7
       EO_SUB_SOURCE: redhat-operators
-      SKIP_TESTS: tests/e2e/instrumentation-sdk tests/e2e/instrumentation-go tests/e2e/instrumentation-apache-multicontainer
-        tests/e2e/instrumentation-apache-httpd tests/e2e/targetallocator-features
-        tests/e2e/prometheus-config-validation tests/e2e/smoke-targetallocator
+      SKIP_TESTS: tests/e2e/targetallocator-features tests/e2e/prometheus-config-validation
+        tests/e2e/smoke-targetallocator tests/e2e-instrumentation/instrumentation-apache-httpd
+        tests/e2e-openshift/otlp-metrics-traces
```
Comment (iblancasa) [inline ci-operator/config/openshift/open-telemetry-opentelemetry-operator/openshift-open-telemetry-opentelemetry-operator-main__ocp-4.14-amd64.yaml:83]: Why do we skip this test?

**Review by iblancasa — APPROVED** [review]

File: ci-operator/config/openshift/open-telemetry-opentelemetry-operator/openshift-open-telemetry-opentelemetry-operator-main__ocp-4.14-amd64.yaml (hunk 1 above)
Comment (IshwarKanse) [reply ci-operator/config/openshift/open-telemetry-opentelemetry-operator/openshift-open-telemetry-opentelemetry-operator-main__ocp-4.14-amd64.yaml:83]: The otlp metrics one I have fixed upstream but our forked repos need to sync for us to enable it. The rest are failing, I’ll debug them next week.

**Pull request merged**